monkeytype/cli.py

```diff
@@ -95,7 +95,8 @@ def get_stub(args: argparse.Namespace, stdout: IO, stderr: IO) -> Optional[Stub]
     rewriter = args.config.type_rewriter()
     if args.disable_type_rewriting:
         rewriter = NoOpRewriter()
-    stubs = build_module_stubs_from_traces(traces, args.include_unparsable_defaults, rewriter)
+    stubs = build_module_stubs_from_traces(traces, args.include_unparsable_defaults,
```

Comment:
time to switch to keyword args here to avoid future bugs related to mis-ordered arguments. Also, minor nit but use this indentation style:

```
stubs = build_module_stubs_from_traces(
    traces,
    include_unparsable_defaults=args.include_unparsable_defaults,
    ignore_existing_annotations=args.ignore_existing_annotations,
    rewriter=rewriter,
)
```

Reply:
fixed.